sync3/conn.go

```diff
@@ -176,7 +182,10 @@ func (c *Conn) OnIncomingRequest(ctx context.Context, req *Request) (resp *Respo
 			if isSameRequest {
 				// this is the 2nd+ time we've seen this request, meaning the client likely retried this
 				// request. Send the response we sent before.
-				logger.Trace().Int64("pos", req.pos).Msg("returning cached response for pos")
+				logger.Trace().Int64("pos", req.pos).Msg("returning cached response for pos, with delay")
```

Comment:
Is there enough info here to see which user/device is causing the spamming?

Reply:
Not on this log line, but surrounding log lines yes.